extract.rb

```diff
@@ -1,6 +1,19 @@
 require 'open-uri'
 
-file = open(ARGV[0]).read
+target = ARGV[0]
+
+if target.nil? or target.empty?
+   puts "Looks like you forgot to supply file path or Url to the target. Kindly specify the target "
```

Comment:
Let's just specify the usage of the command when this happens:

```
puts 'USAGE: ruby extract.rb <file or URL>'
```

Reply:
Sure.